src/pages/SubmissionHistory/SubmissionHistory.js

```diff
@@ -123,16 +134,26 @@ export const SubmissionHistory = () => {
                 paddingTop: '1em'
               }),
               columns: [
-                ...isActionMenuEnabled() ?
+                ...getConfig().isActionMenuEnabled ?
                   [{
                     size: { basis: 100, grow: 0 },
                     field: 'actions',
-                    headerRenderer: () => h(TextCell, ['Actions']),
-                    cellRenderer: () => {
-                      return div(
-                        { style: { textAlign: 'center' } },
-                        [icon('cardMenuIcon', { size: 24, onClick: () => { window.alert('TODO: go to actions menu') } })]
-                      )
+                    headerRenderer: () => h(TextCell, {}, ['Actions']),
+                    cellRenderer: ({ rowIndex }) => {
+                      return h(MenuTrigger, {
+                        'aria-label': 'Action selection menu',
+                        content: h(Fragment, [
+                          h(MenuButton, {
+                            style: { fontSize: 15 },
+                            onClick: () => { cancelRunSet(paginatedPreviousRunSets[rowIndex].run_set_id) }
```

Comment:
Will this allow user to abort a submission even if it is in a terminal state?

Reply:
I think so, although nothing happens. I'll add a check in `cancelRunSet` that'll notify a user that they cannot cancel a terminal run set.